compiler/commands.nim

```diff
@@ -396,6 +396,8 @@ proc processSwitch*(switch, arg: string, pass: TCmdLinePass, info: TLineInfo;
   var
     key, val: string
   case switch.normalize
+  of "input", "i": # PRTEMP
```

Comment:
Sorry for the noob question, but what is "PRTEMP"?

Reply:
unrelated commit from another PR ended up by accident; 
just a token I use to remind myself that something must be addressed before a PR is ready and I usually check for this, but somehow i forgot here